examples/go-migrations/custom.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	// Import your migrations directory here
+	_ "github.com/pressly/goose/examples/go-migrations/migrations"
+	// In your repo, replace "goose" line above with the path to goose:
+	"github.com/pressly/goose"
```

Comment:
Namespacing, I would comment out the migration import after previously suggested change